api/client/cli.go

```diff
@@ -93,10 +93,11 @@ func (cli *DockerCli) Subcmd(name, signature, description string, exitOnError bo
 	flags := flag.NewFlagSet(name, errorHandling)
 	flags.Usage = func() {
 		options := ""
+		tempSignature := " " + signature
```

Comment:
@duglin , if we just change the option , but not the signature, it will looks like:
![usage](https://cloud.githubusercontent.com/assets/9923237/6443832/be2f310a-c134-11e4-98bc-1017270a24f9.png)
I don't sure weather is any better than the oringal one? how do u think?